slycot/tests/test.py

```diff
@@ -64,9 +58,5 @@ def test_td04ad_static(self):
             nr,a,b,c,d = transform.td04ad(rc,nin,nout,index,den,num)
 
 
-def suite():
```

Comment:
This is an old artifact, right? Nothing calls this function and it would fail because of undefined `TestConvert`.

Same exists in many other test scripts here.